relion/convert/convert31.py

```diff
@@ -632,6 +632,10 @@ def _get(label):
         M[:3, 3] = shifts[:3]
         particle.getTransform().setMatrix(M)
 
+    def setPixelSize(self, pixelSize):
```

Comment:
not accepted. Please do not modify working code or open an issue if there's a problem

Reply:
I do not really need this function so I delete it